src/pages/Card.tsx

```diff
@@ -0,0 +1,88 @@
+import { Flex, Box, useColorModeValue, Button } from "@chakra-ui/react";
+import styled from "styled-components";
+import { useState } from "react";
+
+//custom components with styled components
+const Middle = styled.div`
```

Comment:
We should consider moving these in a styled folder in the future

Reply:
Alright will make changes